saleor/settings.py

```diff
@@ -175,6 +176,14 @@ def get_bool_from_env(name, default_value):
 # Make this unique, and don't share it with anybody.
 SECRET_KEY = os.environ.get("SECRET_KEY")
 
+# Add additional password algorithms that can be used
+# The password hashes not using the preferred algorithm (first in the list)
+# will be automatically converted upon user login
```

Comment:
To be honest, I have a problem with understanding this description. Could you improve that? There is no dot between the first and second lines. I also do not understand what will be converted and into what.

Reply:
Does that make more sense now?
```suggestion
# Additional password algorithms that can be used by Saleor.
# The first algorithm defined by Django is the preferred one; users not using the
# first algorithm will automatically be upgraded to it upon login